app/pages/system/networking/IpPoolPage.tsx

```diff
@@ -75,6 +77,22 @@ export function IpPoolPage() {
       <PageHeader>
         <PageTitle icon={<Networking24Icon />}>{pool.name}</PageTitle>
       </PageHeader>
+      <PropertiesTable.Group className="mb-12">
+        <PropertiesTable>
+          <PropertiesTable.Row label="Description">
+            {pool.description || <EmptyCell />}
+          </PropertiesTable.Row>
+          <PropertiesTable.Row label="ID">{pool.id}</PropertiesTable.Row>
+        </PropertiesTable>
```

Comment:
Silo, the most similar existing one of these, puts ID first, and I think that looks better when the description is empty or short.

<img width="1191" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/9048ba79-e23f-49a4-9b94-5338ba451cc1">